src/views/pages/candidates/candidates-search/SearchCandidatesPage.tsx

```diff
@@ -0,0 +1,97 @@
+
+    const onViewCandidateDetails = (e: MouseEvent<HTMLButtonElement>) => {
+        const { id } = e.currentTarget;
+        history.push(`/admin${CANDIDATE_DETAILS}/${id.toLowerCase()}`);
+    };
+
+    const onExportCandidate = (selectedCandidates: ICandidate[]) => {
+        console.log("export", selectedCandidates)
+    }
+
+    const onDeleteCandidates = (selectedCandidates: ICandidate[]) => {
+        console.log("delete", selectedCandidates)
+    }
+
+    const onCandidateRatingChange = (newRating: number, reqId: string) => {
+        updateCandidate({
+            reqId: reqId,
+            body: { rating: newRating },
+        });
+        console.log("update rating", newRating, reqId)
+    }
+
+    const onSearchCandidates = (filters: ICandidateFilters) => {
+        console.log("Search filters", filters)
+    }
+
+    return (
+        <>
+            <GradientEmptyHeader/>
+            <Container className="mt--6" fluid>
+                <Row>
+                    <Col>
+                        <SearchFilterPanel
+                            title="Search Candidates"
+                            statusSelectOptions={getCandidatesSelectStatus}
+                            callback={onSearchCandidates}
+                        />
+                    </Col>
+                </Row>
+                <Row>
+                    <Col>
+                        <Card>
+                            <CardHeader className="d-flex justify-content-between">
+                                <Row>
+                                    <Col md="12">
+                                        <h3 className="mb-0">Candidates</h3>
+                                        <p className="text-sm mb-0">
+                                            Applicants
+                                        </p>
+                                    </Col>
+                                </Row>
+                            </CardHeader>
+                            <ReactTable
+                                data={mockedCandidates}
+                                selectElements={[
+                                    <TableSelectButton
+                                        title="Export"
+                                        callback={onExportCandidate}
+                                    />,
+                                    <TableSelectButton
+                                        title="Delete"
+                                        color="danger"
+                                        callback={onDeleteCandidates}
+                                    />,
+                                ]}
+                                columns={candidatesTableColumns({
+                                    updateRatingHandler: onCandidateRatingChange,
+                                    onDetailsButtonClick: onViewCandidateDetails,
+                                })}
+                            />
+                        </Card>
+                    </Col>
```

Comment:
You can check if this could be extracted in a dedicated panel with different configuration to be shared among Candidate, CV, Interview and Offer page.
I would expect table to look exactly same on columns.
What probably is changing are buttons, list of available status and panel names